app/components/task_list_item_references_component.html.erb

```diff
@@ -1,7 +1,11 @@
 <div class="app-task-list__content">
   <% if FeatureFlag.active?(:reference_selection) %>
     <p class="govuk-body">
-      <%= govuk_link_to t('page_titles.references_review'), references_path, 'aria-describedby': "#{t('page_titles.references_review').parameterize}-badge-id" %>
+      <% if references.count > 1 %>
```

Comment:
Does reference state matter here?

If so, perhaps this should be:
`references.first.application_form.minimum_references_available_for_selection?`

If not, we could just go with the following, and the commit message reworded accordingly:
`if references.present?`

Reply:
Whatever we decide here in terms of what link should show when, it looks like the entire if statement [here](https://github.com/DFE-Digital/apply-for-teacher-training/pull/4894/commits/f452692b25845cf7c82dc67f0f0866f72af41d6e#diff-30c2dce8fba5f45309b4ca92ab0c7886e6a7764a0a91e314dcd3dee11d84998aR57) can be absorbed by this component.

Reply:
Note: Have added this comment to the trello card to-do list and will pick it up once we've finished the initial round of user testing

Reply:
Note for posterity: the way this link behaves will likely change following user testing. Not carding anything up at this stage, will revisit if there's anything that needs updating.